Misc/NEWS.d/next/IDLE/2018-11-11-17-13-50.bpo-34864.cw0PvO.rst

```diff
@@ -0,0 +1 @@
+On macOS, warn if "Prefer tabs when opening documents" is set to "Always".
```

Comment:
For the idlelib/News.txt entry, I am thinking about the more specific
```
When opening IDLE with Shell on macOS, warn if System
Preference "Prefer tabs when opening documents" is set to "Always".
```
Do what you like best for the blurb.

Reply:
The issue I addressed is when the check is made, which appears to at at startup if Shell is opened and no code is run by command line options.  Where messages go is a different issue.